src/main/java/ru/timur/learning/configuration/ApplicationInitializer.java

```diff
@@ -1,27 +1,21 @@
 package ru.timur.learning.configuration;
 
-import javax.servlet.ServletContext;
-import javax.servlet.ServletRegistration;
-import org.springframework.web.WebApplicationInitializer;
-import org.springframework.web.context.ContextLoaderListener;
-import org.springframework.web.context.support.AnnotationConfigWebApplicationContext;
-import org.springframework.web.context.support.GenericWebApplicationContext;
-import org.springframework.web.servlet.DispatcherServlet;
+import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;
 
-public class ApplicationInitializer implements WebApplicationInitializer {
```

Comment:
In this file I didn't change anything - just used out-of-the-box AbstractAnnotationConfigDispatcherServletInitializer which does exactly the same work you did manually before